packages/application-shell/src/components/navbar/use-navbar-state-manager.ts

```diff
@@ -34,22 +34,29 @@ type HookProps = {
   environment: TApplicationContext<{}>['environment'];
   project: TFetchProjectQuery['project'];
 };
+type TMousePosition = {
+  clientX: number;
+  clientY: number;
+};
```

Comment:
nit: I think we define this type twice in this PR

Reply:
Updated here: https://github.com/commercetools/merchant-center-application-kit/pull/3536/commits/b662e8f9258af8523279fa2a301a35959e6a6b33